flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/DecimalTypeTest.scala

```diff
@@ -704,13 +704,13 @@ class DecimalTypeTest extends ExpressionTestBase {
       'f42 % 'f41,
       "f42 % f41",
       "mod(f42, f41)",
-      "2.00")
+      "2.0000")
```

Comment:
I do not see any change can make this result different

Reply:
See CALCITE-3435.